index.html

```diff
@@ -15,50 +15,63 @@
+                <span class="sr-only">Toggle navigation</span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+            </button>
+        </div>
+
+        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+
+            <ul class="ulmenu nav navbar-nav">
+                <li><a href="#Mainpage" data-title="Komunikacja Miejska Gdańsk"><i class=" imagepad fa fa-connectdevelop" aria-hidden="true"></i> KomunikacjaMiejskaGdańsk</a></li>
+                <li class="dropdown">
+                    <a href="#Timetable" class="" data-title="Rozkład Jazdy"><i class="fa fa-calendar"
+                                                                                       aria-hidden="true"></i> Rozkład Jazdy</a>
+                    <div class="dropdown-content btncollapse">
+                        <a href="#Timetable/Buses" data-title="Autobusy"><i class="fa fa-bus" aria-hidden="true"></i>
+                            Autobusy</a>
+                        <a href="#Timetable/Trams" data-title="Tramwaje"><i class="fa fa-subway" aria-hidden="true"></i>
+                            Tramwaje</a>
+                        <a href="#Timetable/SKM" data-title="SKM"><i class="fa fa-train" aria-hidden="true"></i> SKM</a>
+                        <a href="#Timetable/PKM" data-title="PKM"><i class="  fa fa-train" aria-hidden="true"></i> PKM</a>
+                    </div>
+                </li>
+                <li><a href="#Navigate" data-title="Wyznacz trasę"><i class="imagepad fa fa-street-view" aria-hidden="true"></i>Wyznacz trasę</a></li>
+                <li><a href="#Info" data-title="Informacje o komunikacji"><i class="imagepad fa fa-info" aria-hidden="true"></i>Informacje o komunikacji</a></li>
+                <li><a href="#Game" data-title="Bus trip"><i class="imagepad fa fa-gamepad" aria-hidden="true"></i> Zagraj</a></li>
+                <li><a href="#Contact" data-title="Kontakt"><i class="imagepad fa fa-envelope-o" aria-hidden="true"></i> Kontakt</a></li>
+                <!--social media-->
+            </ul>
+        </div>
     </div>
 </nav>
 <header id="Mainpage">
     <!--Background-->
-    <div class="scroll-bg"></div>
-    <!--The End of Background-->
-    <!--Preview-->
-    <div class="preview animated rotateInDownLeft">
-        <div class="col-xs-8 col-xs-offset-2 preview-img">
-            <img src="Assets/Photos/font-awesome-zajawka.png"
-                 alt="Connect-develop img" class="img-responsive">
-        </div>
+    <div class="scroll-bg">
+
+
         <div class="preview-h1 col-xs-8 col-xs-offset-2">
-            <h1>Twój przewodnik po komunikacji miejskiej w Gdańsku</h1>
+
+            <h1 class="Title">Twój przewodnik po komunikacji miejskiej w Gdańsku</h1>
```

Comment:
At lower resolutions the heading "Twój przewodnik..." covers the logo. I think we could put the heading into a separate div and make sure it's always above the logo.